src/App.js

```diff
@@ -20,55 +20,66 @@ function Board({ xIsNext, squares, onPlay }) {
     } else {
       nextSquares[i] = "O";
     }
-    onPlay(nextSquares);
+    onPlay(nextSquares, i);
   }
-  const winner = calculateWinner(squares);
+
+  const winnerData = calculateWinner(squares);
   let status;
-  if (winner) {
-    status = "Winner: " + winner;
+  if (winnerData && winnerData.winner) {
+    status = "Winner: " + winnerData.winner;
+  } else if (currentMove === 9 && !winnerData) {
+    status = "Draw";
   } else {
     status = "Next player: " + (xIsNext ? "X" : "O");
   }
   
-function Row(n) {
-  const row = [];
-  for(let i = n; i < n + 3; i++) {
-    row.push(<Square value={squares[i]} onSquareClick={() => handleClick(i)}/>);
+  function Row(n) {
+    const row = [];
+    let a, b, c;
+    if (winnerData) { [a, b, c]= winnerData.winnerLine;};
+ 
+    for(let i = n; i < n + 3; i++) {
+      if(winnerData && (i === a || i === b || i === c)) {
+        row.push(<Square className="winner" value={squares[i]} onSquareClick={() => handleClick(i)} />);
```

Comment:
Correct an error from the console with unique keys.

Reply:
Done.